__tests__/helpers/setup-oclif-config.ts

```diff
@@ -0,0 +1,42 @@
+import type { Command as OclifCommand } from '@oclif/core';
+
+import path from 'node:path';
+
+import { Config } from '@oclif/core';
+import { captureOutput } from '@oclif/test';
+
+/**
+ * Used for setting up the oclif configuration for simulating commands in tests.
+ * This is a really barebones approach so we can continue using vitest + nock
+ * how we want to.
+ *
+ * @see {@link https://github.com/oclif/test}
+ * @see {@link https://oclif.io/docs/testing}
+ */
+export default function setupOclifConfig() {
+  // https://stackoverflow.com/a/61829368
+  const root = path.join(new URL('.', import.meta.url).pathname, '.');
+
+  return Config.load({
+    root,
+    version: '7.0.0',
+  });
+}
+
+export function runCommand<T extends typeof OclifCommand>(Command: T) {
+  return async function runCommandAgainstArgs(args?: string[]) {
+    const oclifConfig = await setupOclifConfig();
+    // @ts-expect-error this is the pattern recommended by the @oclif/test docs.
+    // Not sure how to get this working with type generics.
+    return captureOutput<string>(() => Command.run(args, oclifConfig), { testNodeEnv: 'rdme-test' }).then(
+      ({ error, result }) => {
+        if (error) {
```

Comment:
Sucks that `@oclif/test` seems to clobber the `SyntaxError` errors that `prepareOas` throws.

<img width="587" alt="Screenshot 2024-11-17 at 12 05 42 PM" src="https://github.com/user-attachments/assets/6a2dd393-d99a-4809-b403-f070918b79e5">

Reply:
Oh looks like you PR'd a fix to them for this? https://github.com/oclif/test/pull/652

Reply:
yep exactly

Reply:
fixed 🥹 https://github.com/readmeio/rdme/pull/1068/commits/6f858aab220552652a46852a85dc1fe58c7b5600